src/uct/ib/rc/base/rc_ep.h

```diff
@@ -57,6 +57,11 @@ struct uct_rc_ep {
 UCS_CLASS_DECLARE(uct_rc_ep_t, uct_rc_iface_t*);
 
 
+typedef struct uct_rc_ep_address {
+    uint8_t             qp_num[3];
+} UCS_S_PACKED ucp_rc_ep_address_t;
```

Comment:
IB QP number is 24 bits, so we save one byte by using array of 3 bytes instead of uint32_t